zero/vlasov_lte_moments.c

```diff
@@ -212,11 +249,21 @@ gkyl_vlasov_lte_moments_release(gkyl_vlasov_lte_moments *lte_moms)
   if (lte_moms->model_id == GKYL_MODEL_SR) {
     gkyl_array_release(lte_moms->M0_minus_V_drift_dot_M1i);
     gkyl_array_release(lte_moms->GammaV_inv);
-    gkyl_array_release(lte_moms->GammaV2);    
+    gkyl_array_release(lte_moms->GammaV2);  
+    gkyl_dg_updater_moment_release(lte_moms->Pcalc);  
+  }
+  else if (lte_moms->model_id == GKYL_MODEL_CANONICAL_PB) {
+    gkyl_array_release(lte_moms->h_ij_inv);
+    gkyl_array_release(lte_moms->det_h);
+    gkyl_array_release(lte_moms->pressure_tensor);
+    gkyl_dg_updater_moment_release(lte_moms->M2ijcalc);
```

Comment:
I think we should either call the needed "M2" calculator Pcalc for all of these quantities (so there's not a distinct M2ijcalc for the canonical PB case) or we should have a different name for the needed 2nd moment updater in all these cases (so for the non-relativistic, non-PB case, we should call it M2calc instead of Pcalc). 

Up to you on whether you want to re-use Pcalc in all three cases or call is something different and release it specifically in all three cases.

Reply:
Fixed in commit https://github.com/ammarhakim/gkylzero/commit/cd5dd0f23cdbc04e19cc228ce3e8d532a945dddc